wger/core/views/user.py

```diff
@@ -540,3 +547,180 @@ def get_context_data(self, **kwargs):
             context['object_list']['members']
         }
         return context
+def oauthLogin(request, platform):
+    '''
+        Social networks login
+    '''
+    authorize_url = '{}/en/user/login/oauth/facebook/authorize'.format(os.getenv('SITE_URL'))
```

Comment:
why is this outside of a condition?

Reply:
Yeah, It should be inside the condition. I missed that. Thanks.